stubs/parsimonious/parsimonious/__init__.pyi

```diff
@@ -1,8 +1,8 @@
 from parsimonious.exceptions import (
-    BadGrammar as BadGrammar,
+    BadGrammar as BadGrammer,
     IncompleteParseError as IncompleteParseError,
     ParseError as ParseError,
     UndefinedLabel as UndefinedLabel,
 )
-from parsimonious.grammar import Grammar as Grammar, TokenGrammar as TokenGrammar
+from parsimonious.grammar import Grammar as Grammar, TokenGrammar as TokenGrammer
```

Comment:
Aren't these changes here incorrect? 🤨

Reply:
? How so.... This was copied 1:1 from the file

Reply:
Oh, you mean the spelling. Yes, that's a downside of today's tooling, the `as` don't offer spell check. I'll fix that

Reply:
`BadGrammar` appears to be correctly spelled as "BadGrammar" on both typeshed `master` and parsimonious `master` :) Same with `TokenGrammar` -- it's "Grammar", not "Grammer". But your PR introduces two typos here, spelling them as "Grammer" rather than "Grammar"

https://github.com/erikrose/parsimonious/blob/0181ef6ad08c03544ea43d351bfc249834ed81e7/parsimonious/__init__.py#L8-L9

Reply:
Yeah, I noticed what you meant after. That's a side effect of re-typing each file by hand in this case to understand what the changes looked like. VSCode doesn't offer any kind of suggestion or support for re-export naming, which leads to typos like this. One of my biggest pet peeves about python 🤷‍♂️